metaflow/plugins/__init__.py

```diff
@@ -251,3 +251,6 @@ def get_plugin_cli():
     except KeyError:
         pass
 del globals()["_n"]
+
+# Checkpoints
+from .frameworks.checkpoints import get_checkpoint_paths, CheckpointPaths
```

Comment:
Is this still needed? It looks like you added the same to `metaflow/__init__.py` so just double checking.

Reply:
removed!